create_infotexts.py

```diff
@@ -435,6 +437,15 @@ def process_region_addition_to_description(region_str, country_str):
 
         return region_addition
 
+    def add_catch_all_category(self, item):
+        """"
+        Populate self.content_cat with commons category, if present.
```

Comment:
I meant that you could add this instead of the `None` I would still keep the original first line of the docstring though.

Also there should be no empty lines towards the end of the docstring (just one between the first line and the rest.